libraries/chain/include/golos/chain/worker_objects.hpp

```diff
@@ -126,6 +129,15 @@ namespace golos { namespace chain {
                     worker_techspec_object,
                     member<worker_techspec_object, account_name_type, &worker_techspec_object::worker_proposal_author>,
                     member<worker_techspec_object, shared_string, &worker_techspec_object::worker_proposal_permlink>>,
+                composite_key_compare<
+                    std::less<account_name_type>,
+                    chainbase::strcmp_less>>,
+            ordered_non_unique<
```

Comment:
This is index is unique. 
We can have multiple tspecs per one proposal.
But one post should be attached to one tspec.
And one post should be attached to one work result.